rpc/proto/mayastor.proto

```diff
@@ -205,6 +207,11 @@ message StartRebuildRequest {
   string uri = 2;   // uri of the child to be rebuilt
 }
 
+message StopRebuildRequest {
+  string uuid = 1;  // uuid of the nexus
+  string uri = 2;   // uri of the child to be rebuilt
```

Comment:
copy-paste error in comment for the uri (and the same for the uri in rebuild-state-request)

Reply:
Amended comments.